<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleConfigure</class>
 <widget class="QWidget" name="StyleConfigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>250</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxBackground">
        <property name="title">
         <string>Background</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonBackgroundGreen">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBackgroundBlue">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBackgroundRed">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBackgroundYellow">
            <property name="text">
             <string>Yellow</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBackgroundGrey">
            <property name="text">
             <string>Grey</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBoxForeground">
        <property name="title">
         <string>Foreground</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButtonForegroundBlue">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonForegroundRed">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonForegroundYellow">
            <property name="text">
             <string>Yellow</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonForegroundBlack">
            <property name="text">
             <string>Black</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonForegroundWhite">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBoxFont">
        <property name="title">
         <string>Font</string>
        </property>
        <widget class="QCheckBox" name="checkBoxBold">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Bold</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxItalic">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Italic</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelSize">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditSize">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
